config.toml.dist

```diff
@@ -1,26 +1,53 @@
+[server]
+# IP address to bind to - Default: "127.0.0.1"
 listen_ip = "127.0.0.1"
+
+# Port to bind to - Default: 3333
 listen_port = 3333
-base_url = "https://portier.example.com"
-token_validity = 600
 
-[[keys]]
-file = "private.pem"
+# Server's user-facing URL - Default: (none)
+public_url = 
+
+
```

Comment:
Spacing is odd here. Think this'd look better if it didn't have newlines between individual fields, but just one between each section?